lib/mittsu/math/euler.rb

```diff
@@ -9,6 +7,7 @@ class Euler
 
     def initialize(x = 0.0, y = 0.0, z = 0.0, order = DefaultOrder)
       @x, @y, @z, @order = x.to_f, y.to_f, z.to_f, order
+      @on_change_callback = false
```

Comment:
Since `@on_change_callback` is normally a proc, I think making it default to `nil` instead would be more appropriate.
```suggestion
      @on_change_callback = nil
```

Reply:
Sure, that makes sense.